sys/compat/freebsd32/freebsd32_systrace_args.c

```diff
@@ -1079,7 +1079,7 @@ systrace_args(int sysnum, void *params, uint64_t *uarg, int *n_args)
 		iarg[a++] = p->msqid; /* int */
 		uarg[a++] = (intptr_t)p->msgp; /* void * */
 		uarg[a++] = p->msgsz; /* size_t */
-		iarg[a++] = p->msgtyp; /* int32_t */
+		uarg[a++] = (intptr_t)p->msgtyp; /* int32_t */
```

Comment:
There is a lot of noise in this file.  The original type of `msgtyp` in syscalls.master is `long`.  It's not clear to me why changing `abi_intptr_t` is affecting `long`.